easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-20.10-iimpi-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,61 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+name = 'OpenMolcas'
+version = '20.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://gitlab.com/Molcas/OpenMolcas"
+description = "OpenMolcas is a quantum chemistry software package"
+
+toolchain = {'name': 'iimpi', 'version': '2020a'}
+
+source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
+sources = ["OpenMolcas-v%(version)s.tar.gz"]
+
+# This patch avoids a forced extra copy of 'pymolcas' script
+# in a local folder of the user doing the build
+# The 'make install' process will already install it in
+# %(installdir)s/sbin/
+patches = ["OpenMolcas-pymolcas_only_in_ebprefix.patch"]
+
+checksums = [
+    # OpenMolcas-20.10.tar.gz
+    '73f73beaebbcff5d02b5bb0748d07c5b8e81784fae6122729c7e7aeed730fe42',
+    # OpenMolcas-pymolcas_only_in_ebprefix.patch
+    '56165ab4a752ef7cdbe3340c8c922fca04510e8ee2e4024ea654e6bcd83198f8'
+]
+
+dependencies = [
+    ('GlobalArrays', '5.8'),
+    ('HDF5', '1.12.0'),
+    ('Python', '3.8.2'),
+    ('imkl', '2020.1.217'),
+]
+
+builddependencies = [('CMake', '3.16.4')]
+
+separate_build_dir = True
+
+preconfigopts = 'GAROOT=$EBROOTGLOBALARRAYS'
+configopts = '-DLINALG=MKL -DMKLROOT=$EBROOTIMKL/mkl -DMPI=ON -DGA=ON -DBUILD_SHARED_LIBS=ON -DTOOLS=ON -DFDE=ON'
+
+modextravars = {
+    'MOLCAS': '%(installdir)s',
+    'MOLCAS_DRIVER': '%(installdir)s/sbin/pymolcas',
+    'MOLCAS_SOURCE': '%(installdir)s',
+}
+
+sanity_check_paths = {
+    'files': ['lib/libmolcas.so', 'bin/alaska.exe', 'bin/casvb.exe', 'bin/cht3.exe', 'bin/espf.exe', 'bin/gateway.exe',
+              'bin/gugaci.exe', 'bin/last_energy.exe', 'bin/mbpt2.exe', 'bin/mcpdft.exe', 'bin/mrci.exe',
+              'bin/qmstat.exe', 'bin/rassi.exe', 'bin/seward.exe', 'bin/vibrot.exe', 'bin/averd.exe', 'bin/ccsdt.exe',
+              'bin/cpf.exe', 'bin/expbas.exe', 'bin/genano.exe', 'bin/gugadrt.exe', 'bin/localisation.exe',
+              'bin/mckinley.exe', 'bin/motra.exe', 'bin/numerical_gradient.exe', 'bin/quater.exe', 'bin/rpa.exe',
+              'bin/slapaf.exe', 'bin/caspt2.exe', 'bin/chcc.exe', 'bin/dynamix.exe', 'bin/ffpt.exe', 'bin/guessorb.exe',
+              'bin/guga.exe', 'bin/loprop.exe', 'bin/mclr.exe', 'bin/mpprop.exe', 'bin/parnell.exe', 'bin/rasscf.exe',
+              'bin/scf.exe', 'bin/surfacehop.exe', 'sbin/pymolcas', 'sbin/help_basis'],
```

Comment:
@nordmoen Ideally these are sorted alphabetically (and we usually only check for a couple of "key" files, not *everything* that should be there.

We also try to avoid hardcoding `.so` (which is specific to Linux), and we use `'lib/libmolcas.%s' % SHLIB_EXT` instead.

I'm aware you inherited this from the existing `OpenMolcas` easyconfig file, but maybe now is a good time to clean it up?

Reply:
I have never used the program so I don't quite know what the key files are. From the `README` of the project the `sbin/pymolcas` is the main entry point. Would you like to just leave it at `['lib/libmolcas.%s' % SHLIB_EXT, 'sbin/pymolcas']`?

Reply:
Definitely those two, and I would include two or more addition binaries (just pick them randomly)

Reply:
Perfect, thanks!